index.html

```diff
@@ -15,6 +15,14 @@ <h2>The World's Foremost Open Source Grand Strategy Game</h2>
     <br>
     <span class="download"><a href="{{ '/download/' | prepend: site.baseurl }}" class="button">Download TripleA</a></span>
       <p>100% open source, community run, free to play game</p>
-      <p>Multiplayer and single player game options</p>
+      <p>Play live online via lobby, play by email or versus AI</p>
     <a href="{{ '/maps/' | prepend: site.baseurl }}"><img src="./images/triplea_promo.png" alt="Screenshot of TripleA" width="1059" height="798"></a>
  </div>
+
+<h2> Software Licensing </h2>
+
+<p>
+TripleA is released under the [GNU General Public License](https://www.gnu.org/licenses/old-licenses/gpl-2.0.html) (GPL); either version 2
```

Comment:
@DanVanAtta Sorry, missed this during the review...  This is a Markdown link, not an HTML link.

Reply:
#289